scale/dev/launch_central.sh

```diff
@@ -2,6 +2,7 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
 
+export ROX_SCANNER_V4=false
```

Comment:
Can we give the cluster that runs ACS during the k6 tests more oomph? Since Scanner v4 becomes the default, we should include that in this test IMO (even if it is only an optional test).